libs/form-elements/src/NumberInput.tsx

```diff
@@ -92,28 +92,34 @@ export type NumberInputProps = {
 } & Omit<NumberFormatProps, NumberFormatOnlyPropsUnion>;
 
 export default function NumberInput({ name, onChange, onBlur, ...props }: NumberInputProps) {
```

Comment:
cant you do the same thing here as on the Date ? in case of the missing intl check if user has manually provided decimal and thousand separators as props?